addons/binding/org.openhab.binding.konnected/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+
+  <parent>
+    <artifactId>pom</artifactId>
+    <groupId>org.openhab.binding</groupId>
+    <version>2.4.0-SNAPSHOT</version> 
+  </parent>
+
+  <artifactId>org.openhab.binding.konnected</artifactId>
+  <packaging>eclipse-plugin</packaging>
+
+  <name>Konnected Binding</name>
+  <profiles>
+  <profile>
```

Comment:
These profiles should be removed since the profiles inherited from parent projects should suffice.